contrib/ansible/xsnippet-api.conf.j2

```diff
@@ -35,7 +35,102 @@ connection = mongodb://db:27017/xsnippet
 ;        python
 ;        javascript
 ;
-; By default, constraint is not defined so any syntax is accepted.
+
+syntaxes =
+    actionscript
```

Comment:
Is this a part of [codemirror syntaxes](https://github.com/codemirror/CodeMirror/tree/master/mode) ? :) Since we are going to use this library as both code editor and syntax highlighter, I'd suggest to use its mode as basis.

Reply:
Ok, will do!